### DISCUSSION TITLE
Do bevel support EKS k8s v1.25?

### DISCUSSION THREAD

**Discussion by cclchd-soundaryaayyappan** [hyperledger-labs/blockchain-automation-framework#2203] [Q&A] [answered]
I have a running EKS cluster with k8s v1.22. I am going to upgrade the version of the cluster to v1.25.

I have deployed fabric network on the cluster using bevel release v0.11.0.0. So, now if I update the k8s version of the EKS cluster, would I be able to use the same bevel version or any other latest release of bevel to do fabric operations on the cluster?

Please post your responses asap. Thanks in Advance!

**Comment by suvajit-sarkar** [comment, marked as answer]
Bevel currently supports Kubernetes v1.22 and we are in the process of adding support for v1.23. There will be changes required on v1.25, of which we are not fully aware. The changes needed will be in storage class (v1.23+ required csi drivers for use of ebs), need of external chaincode as docker is removed in 1.24+ and could be some Kubernetes API upgrades.

**Reply by cclchd-soundaryaayyappan** [reply]
Okay. Thank you for the response @suvajit-sarkar
